src/server/sim-files.js

```diff
@@ -139,7 +139,12 @@ SimulationFiles.prototype._createHostJsFile = function (simulationFilePath, host
 
     var scriptDefs = createScriptDefs(hostType, scriptTypes);
 
-    var b = browserify({ paths: getBrowserifySearchPaths(hostType), debug: true });
+    var b = browserify({
+        paths: getBrowserifySearchPaths(hostType),
+        debug: true,
+        externalRequireName: '_cordovaSimulateRequire'
```

Comment:
Looks like `opts.externalRequireName` is only used if `opts.hasExports` is specified, which we don't. Although looking closer at browserify, it does seem to specify `hasExports` if `exports` is undefined and `opts.entry` is falsy, which looks to be the case at the moment.